scripts/azure-pipelines/install-environment_linux.bash

```diff
@@ -4,13 +4,14 @@
 # Use of this source code is governed by a BSD-style license
 # that can be found in the LICENSE file at the root of the
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
-
+sudo add-apt-repository ppa:beineri/opt-qt-5.12.7-xenial -y
```

Comment:
I'm not entirely sure whether setting the Qt dependency to 5.12 might be a problem. As we are trying to support the latest Ubuntu LTS (which I think has an older Qt version), this might contradict that.

@davidebeatrici what are your thoughts on this?
Iirc we wanted to update Qt to 5.12 anyways (at some point) - at least I think I read that in an issue somewhere...

Reply:
Actually the updater requires at least **Qt 5.6.0**, Can you afford that???

Reply:
Looking at the package list at https://packages.ubuntu.com/xenial/devel/ it seems like ubuntu 16.04 (which is currently the oldest Ubuntu LTS) ships with Qt 5.5.1

As this LTS is supported until beginning of next year, I'd say that 5.6.0 might not be viable until then. If however we can make the build-env so that it requires Qt >= 5.6.0 only if support for the update bridge is enabled, I think we're good :)

Reply:
@probonopd built an AppImage that contains all dependencies in #3959.

If it's confirmed as stable, we can use the latest Qt version without any issues.

Reply:
Even without using that, you can use much newer Qt versions than what comes with xenial itself, e.g., the Qt builds for xenial from https://launchpad.net/~beineri. This is because we bundle a subset of Qt inside the AppImage.

Reply:
Awesome! In that case we should always use the latest Qt version, because LTS will not be freely available anymore: https://www.qt.io/blog/qt-offering-changes-2020